yarn.lock

```diff
@@ -2164,7 +2164,12 @@
   dependencies:
     "@types/node" "*"
 
-"@types/node@*", "@types/node@>= 8":
+"@types/node@*":
+  version "13.7.0"
+  resolved "https://registry.yarnpkg.com/@types/node/-/node-13.7.0.tgz#b417deda18cf8400f278733499ad5547ed1abec4"
+  integrity sha512-GnZbirvmqZUzMgkFn70c74OQpTTUcCzlhQliTzYjQMqg+hVKcDnxdL19Ne3UdYzdMA/+W3eb646FWn/ZaT1NfQ==
+
+"@types/node@>= 8":
```

Comment:
roll back change here?

Reply:
I have upgraded @types/node to 13.7 because this version includes syncBuiltinESMExports in module.

Reply:
Which I have used here
https://github.com/facebook/jest/pull/9469/files#diff-5b473ffb7e91f9fc070a6f6025790ef0R918

Reply:
I have reverted changes back. I just used another type

Reply:
oh! I just didn't want 2 versions of it, could have deduped it to the newer ones. Lemme play with it locally. Thanks!